lib/src/item_tags.dart

```diff
@@ -438,9 +439,9 @@ class _ItemTagsState extends State<ItemTags> {
   void _singleItem(DataListInherited dataSetIn, DataList? dataSet) {
     dataSetIn.list!
         .where((tg) => tg != null)
```

Comment:
To avoid use of `!`.
```suggestion
        ..whereNotNull()
```

Reply:
Thanks but my IDE complains when using `whereNotNull`, as if it didn't exist...

Reply:
My bad, it is an extension method from the [collection package](https://pub.dev/packages/collection) and it is not used in this package, I'm sorry. (and sorry for the very long delay)